tasks.c

```diff
@@ -450,7 +450,7 @@ static void prvInitialiseTaskLists( void ) PRIVILEGED_FUNCTION;
  * void prvIdleTask( void *pvParameters );
  *
  */
-static portTASK_FUNCTION_PROTO( prvIdleTask, pvParameters );
+static portTASK_FUNCTION_PROTO( prvIdleTask, pvParameters ) PRIVILEGED_FUNCTION;

```

Comment:
Will this allow people to run privileged code in an idle task hook?

Reply:
Idle task always run privileged and therefore, idle hook also runs privileged. Any code written in the idle hook will always run privileged regardless of the above change. This change only ensures that the idle task's definition is placed in the privileged flash.